### PR TITLE
Capistrano 3

### PR DESCRIPTION
**Pull request by tijsverkoyen**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,70 +0,0 @@
-GEM
```
Comment (Extrapolator214) [inline Gemfile.lock]: Gemfile.lock is mandatory, please run `bundle install` to generate/update it

File: app/config/capistrano/stages/staging.rb [hunk 2]
```diff
@@ -1,33 +1,17 @@
...
-# http://net-ssh.github.io/net-ssh/classes/Net/SSH.html#method-c-start
-#
-# Global options
-# --------------
-#  set :ssh_options, {
-#    keys: %w(/home/rlisowski/.ssh/id_rsa),
-#    forward_agent: false,
-#    auth_methods: %w(password)
-#  }
+### OLD ###
```
Comment (carakas) [inline app/config/capistrano/stages/staging.rb]: does this still have a use?

File: app/config/capistrano/tasks/maintenance.rake [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        RewriteCond %{REQUEST_URI} !.(css|gif|jpg|png)$
+        RewriteCond %{SCRIPT_FILENAME} !maintenance.html
+        RewriteRule ^.*$ /maintenance.html [L]
+      EOF
+      upload! StringIO.new(redirect_to_maintenance), "#{htaccess_path}"
+      execute :chmod, "644 #{htaccess_path}"
+    end
+  end
+
+  task :disable do
```
Comment (carakas) [inline app/config/capistrano/tasks/maintenance.rake]: I would also check here if the symlink is still correct. We now have to fix the symlink sometimes after a failed deploy. But maybe this is not the correct place for that

File: app/config/capistrano/deploy.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
+set :client,  "client"
```
Comment (tijsverkoyen) [inline app/config/capistrano/deploy.rb]: don't harcode these values.

File: app/config/capistrano/tasks/maintenance.rake (hunk 3 above)
Comment (tijsverkoyen) [inline app/config/capistrano/tasks/maintenance.rake]: That issue should be resolved already

**Pull request merged**